app/utils/file/index.ts

```diff
@@ -362,7 +362,7 @@ export function getLocalFilePathFromFile(serverUrl: string, file: FileInfo | Fil
                 }
             }
 
-            return `${FileSystem.cacheDirectory}${server}/${filename}-${hashCode(file.id!)}.${extension}`;
+            return `${FileSystem.cacheDirectory}${server}/${filename.replace(/ /g, '-')}-${hashCode(file.id!)}.${extension}`;
```

Comment:
Out of curiosity, what is this for? I guess it changes any white space by a `-`, but I wonder if it is some kind of standard on mobile, or I am missing something.

Reply:
yeah I'm actually not convinced.. going to revert it